arkouda/strings.py

```diff
@@ -359,8 +359,7 @@ def get_lengths(self) -> pdarray:
 
     def get_bytes(self):
         """
-        Property accessor that returns a pdarray of bytes (uint8) of the
-        corresponding accessed Strings or SegString.
+        Getter for the offsets component (int64 pdarray) of this Strings.
```

Comment:
you changed both to be the docstring of `offsets`
```suggestion
        Getter for the bytes component (uint8 pdarray) of this Strings.
```

Reply:
Updated